src/operation-poller.ts

```diff
@@ -39,6 +41,7 @@ export class OperationPoller<T> {
       name: options.pollerName || "LRO Poller",
       concurrency: 1,
       retries: Number.MAX_SAFE_INTEGER,
+      maxBackoff: options.maxBackoff || DEFAULT_MAX_BACKOFF_MILLIS,
```

Comment:
I don't think it's necessary to include `DEFAULT_MAX_BACKOFF_MILLIS` here. Since `maxBackoff` has a default in the Queue class, we don't need to duplicate that here (which is all that it's doing).

Reply:
Sounds good, done.